func-complete-check/check-receiver/v3.ts

```diff
@@ -0,0 +1,26 @@
+import { Context } from '@azure/functions'
+import { SubmittedCheckMessageV3, ValidateCheckMessageV1, ReceivedCheck } from '../typings/message-schemas'
+import Moment from 'moment'
+import azureStorageHelper from '../lib/azure-storage-helper'
+const tableService = azureStorageHelper.getPromisifiedAzureTableService()
+
+class V3 {
+  async process (context: Context, receivedCheck: SubmittedCheckMessageV3) {
+    const receivedCheckEntity: ReceivedCheck = {
+      PartitionKey: receivedCheck.schoolUUID,
+      RowKey: receivedCheck.checkCode,
+      archive: receivedCheck.archive,
+      checkReceivedAt: Moment().toDate(),
+      checkVersion: +receivedCheck.version
+    }
+    await tableService.insertEntityAsync('receivedCheck', receivedCheckEntity)
+    const message: ValidateCheckMessageV1 = {
+      version: '1',
```

Comment:
Should version be an int?

Reply:
that's what i did at first.  but even with the incoming message set to be of type `ValidateCheckMessageV1` it comes in as a string, probably due to the fact that the message body is a string. i will revisit this in the next PBI